src/OpenTK.Windowing.Desktop/NativeWindowSettings.cs

```diff
@@ -170,9 +171,19 @@ public NativeWindowSettings()
         /// </summary>
         /// <remarks>
         /// Pass <c>-1</c> as the width or height to ignore the respective dimension.
+        /// If you set size limits and an aspect ratio that conflict, the results are undefined.
         /// </remarks>
         public Vector2i MaximumSize { get; set; } = new Vector2i(GLFW.DontCare, GLFW.DontCare);
 
+        /// <summary>
+        /// Gets or sets the initial aspect ratio of the window as a numerator and denominator.
```

Comment:
It's not only the initial aspect ratio, it's locked to this aspect ratio unless you change it later.